### PR TITLE
Big TWO chem nerf

### PR DESCRIPTION
**Pull request by LanCartwright**
Neurotoxin.
Heparin.

:cl: optional name here
balance: Neurotoxin alcohol requires 20 or cycles to stun. No more immediate and uncounterable stuns.
balance: Heparin damage and blood loss heavily reduced, no more instakill syringes that cause 80% blood loss and crit-tier brute damage..
/:cl:

### CODE REVIEW

**Comment by The-clown-known-as-Pepe** [comment]
add heart attack to blackpowder
 👀

**Comment by LanCartwright** [comment]
# No.

**Comment by kevinz000** [comment]
you forgot beepsky smash

**Comment by LanCartwright** [comment]
requires iron, comes next, will consider it.

**Comment by CitrusGender** [comment]
I made an account just for this. For what reason are you doing this? Does medbay not deserve to have some way of clearing rough patients out of medbay? This would leave few alternatives.

**Comment by PraiseRatvar** [comment]
@CitrusGender because stuns are a dull combat mechanic plus its a way better taser

**Comment by LanCartwright** [comment]
@CitrusGender 
Morphine.
Chloral Hydrate.
Neurotoxin poison.
Tirizine.
**Security**

**Comment by CitrusGender** [comment]
@PraiseRatvar I mean, wouldn't that make more sense to just reduce the stuntime with the metabolism rate/reducing the effect time? This just outright removes the stun. I'm all for decreasing the time but this just seems a tad extreme.

**Comment by CitrusGender** [comment]
@LanCartwright 
**Security:**
Literarly any hardsuit
Stun weapon
Stun baton

Morphine is a delayed stun and takes a hell of a lot of time to act. Additionally, I don't know how you can possibly say that securtiy has no way of removing people from places.

**Comment by LanCartwright** [comment]
@CitrusGender 
You do not need an immediate stun.

**Comment by BeeSting12** [comment]
@LanCartwright 
Heparin is fine as is. Don't know about neurotox, but heparin is fine.

**Comment by Lexorion** [comment]
While "uncounterable" stuns are upsetting, it's the only one chemistry has (if I'm not wrong) and it's not too easy to make. The bartender and the chemists can't make the drink alone and it requires co-operation or you have to break in and do it yourself which is a shitty thing to do and it's still risky because it can get you arrested or killed. It is strong but hardsuits completely protect you against the syringes unless they have the special RND ones. Reducing the stun time or increasing the metabolism rate (both maybe) like @CitrusGender suggested would be **far** better. Chloral Hydrate is basically a joke unless you OD someone to kill them and morphine takes too long in my opinion, so please don't just turn it into morphine 2.0. Also what's wrong with black powder?

**Comment by LanCartwright** [comment]
Doesn't matter, a 1 hit kill Syringe is not in the spirit of the game.

Also doesn't matter about cooperation. This happens too often not to mention it's literally ("vodka" = 1, "gin" = 1, "whiskey" = 1, "cognac" = 1, "limejuice" = 1) which isn't hard to get bar limejuice.


You are not getting an instant stun.


Blackpowder explosions.

**Comment by CitrusGender** [comment]
@LanCartwright I mean, does anyone deserve an immediate stun in that case? It sounds like your problem is with stuns in general if that is the case, which is another argument for another day. Look at where this leaves medbay: 0 reliable stuns and CMO has a telescopic baton. Under you logic, they should remove the telescopic baton since they don't deserve immediately stuns. Additionally, that "immediate stun" is really bulky since it's a one shot that needs to be reloaded inbetween each shot. In a normal combat situation, this is unreliable at best and a liability at worst since you're constantly fumbling with your hands. An unreliable weapon that can only shoot once and doesn't penetrate hardsuits is nothing short of something just to remove unruly patients out of medbay. Honestly, I don't even use it when I'm traitor too since it's so bulky in comparison to other items I could pick.

**Comment by LanCartwright** [comment]
You are not getting a Chem based instastun at roundstart. You have Zombie powder.

The combination of a Chem based instastun allows for instakill syringes with no counter. This ends here.

**Comment by Lexorion** [comment]
There are better options for "1 hit kill" syringes which don't even rely on **any** stuns. Neurotoxin alone does not kill anyone and even people who aren't antags can use it in some cases. There are far better ways to kill people as chemist but there are barely any nonlethal ways to disable someone in a fight without killing them, excluding normal stun items of course. This would encourage killing over stunning someone in case bad things happen in medbay which is also dumb. I understand your frustration about stuns and how little you can do about it but "I DED PLS NERF" PRs are bad. Other stuns are literally stronger and even harder to counter. Come on, you are better than that.
And I thought proper explosions take quite the amount of blackpowder? How much do you need to blow someone up? I thought it was more than 50 or something for a small explosion but I'd have to test it I guess.

**Comment by LanCartwright** [comment]
This is a targeted nerf to Neurotoxin, Perflourodecalin, Heparin instakill/sub 60% blood syringes.

You do not need an instant stun in Chem form at roundstart.

This isn't a stun that can be reduced either, which is fundamentally broken.

**Comment by Lexorion** [comment]
You can change the metabolism rate to make it quite a bit weaker but it's not fundamentally broken/ a roundstart stun. You can stun and cuff someone and then kill him with just a single hit and a lot less preparation, I think you are exaggerating quite a bit since in the end, it's a less reliable stun which takes longer to make. I agree with the Heparin changes though.

**Comment by LanCartwright** [comment]
No.

You do not need an instant stun in Chem form at roundstart.

**Comment by CitrusGender** [comment]
I don't know what to say, you keep responding with short sentences that basically just said the same thing as what I am trying to refute. You're not providing any reasoning behind it, you're just saying (continously) that "you don't need it." If that is the case, then nobody needs anything: remove stuns because we don't neccesarily need them. This is just devolved into your opinion. All I can do is respectfully disagree.

**Comment by LanCartwright** [comment]
Here's the reasoning, I'm solving a player issue by fixing the code.

Round after round players would make syringes that would instantly mute and stun an individual and leave them bleeding and taking brute damage. By the time the stun ended, more than 90 brute damage was dealt and more than 40% blood loss occurred and would often instantly Crit or kill someone.

Regardless, chemical stuns with exception of Zombie powder (sleepypen) was delayed, and will stay delayed instead of instant.



Regardless, your points of shitters in medbay is a player issue. Ahelp that or ask sec to deal with it.

**Comment by Lexorion** [comment]
Here's whats going to happen: You'll get hit by one of the other 100 insta kill mixes, a lot of them won't even need stuns. I feel like you are making an issue out of something minor here, atleast with the neurotoxin, since hitting the syringe is far harder than just using any baton. The sleepy pen doesn't even have zombie powder anymore as far as I know. The heparin part of the PR does solve some issues but I'm not sure about the blackpowder and I really disagree with the way you want to change neurotox. There are better ways adjust some of the stronger chems without taking away all of the freedom and creativity.

**Comment by LanCartwright** [comment]
there won't be another instastun muting instakill mix.

**Comment by PopNotes** [comment]
the only good change is neurotoxin, everything else is fucking terrible

also neurotoxin's cycle count needs to be lowered to about ten, but that's not going to happen since lan is absolutely convinced he's right and everyone else is wrong

**Comment by TrickyDickRichard** [comment]
urgay

**Comment by Repukan** [comment]
Leave heparin alone

File: code/modules/reagents/chemistry/reagents/pyrotechnic_reagents.dm [hunk 1]
```diff
@@ -106,13 +106,31 @@
 	metabolization_rate = 0.05
 	taste_description = "salt"
 
+/datum/reagent/blackpowder/on_mob_life(mob/living/M)
+	M.losebreath += 10
+	if(holder.has_reagent("blackpowder"))
```
Comment (Iamgoofball) [inline code/modules/reagents/chemistry/reagents/pyrotechnic_reagents.dm]: Ehh, not really the point here. It's supposed to stick around awhile for trap detonations.

File: code/modules/reagents/chemistry/reagents/pyrotechnic_reagents.dm [hunk 2]
```diff
@@ -106,13 +106,31 @@
...
+	name = "Damp Black Powder"
+	id = "blackpowder"
+	description = "It's all wet and damp now, and unlikely to explode. Very bad for the heart and prevents breathing."
+	reagent_state = LIQUID
+	color = "#000000"
+	taste_description = "salt"
+
+/datum/reagent/dampblackpowder/on_mob_life(mob/living/M)
+	M.losebreath += 2
+	M.visible_message("<span class='userdanger'>[M] clutches their chest and heaves in pain!</span>")
```
Comment (Iamgoofball) [inline code/modules/reagents/chemistry/reagents/pyrotechnic_reagents.dm]: message spam

**Comment by DoctoreMad** [comment]
>its another "Dude who made PR doesn't know how to talk so he just loses his shit in the Conversation tab" episode

Someone change the channel

File: code/modules/reagents/chemistry/reagents/alcohol_reagents.dm [hunk 3]
```diff
@@ -1162,10 +1162,10 @@ All effects don't start immediately, but rather get worse over time; the rate is
 	glass_desc = "A drink that is guaranteed to knock you silly."
 
 /datum/reagent/consumable/ethanol/neurotoxin/on_mob_life(mob/living/carbon/M)
-	M.Knockdown(60, 1, 0)
 	M.dizziness +=6
 	switch(current_cycle)
-		if(15 to 45)
+		if(20 to 45)
+			M.Knockdown(40, 1, 0)
```
Comment (Tokiko1) [inline code/modules/reagents/chemistry/reagents/alcohol_reagents.dm]: Won't this stop causing knockdown after cycle 45?

**Comment by CPTANT** [comment]
Why would you remove mob blackpowder detonations?

That is one of the best applications there is and it is pretty hard to pull off in the first place.

File: code/modules/reagents/chemistry/reagents/pyrotechnic_reagents.dm [hunk 4]
```diff
@@ -106,13 +106,31 @@
 	metabolization_rate = 0.05
 	taste_description = "salt"
 
+/datum/reagent/blackpowder/on_mob_life(mob/living/M)
+	M.losebreath += 10
+	if(holder.has_reagent("blackpowder"))
+		holder.remove_reagent("blackpowder", 10)
+		holder.add_reagent("dampblackpowder", 5)
```
Comment (CPTANT) [inline code/modules/reagents/chemistry/reagents/pyrotechnic_reagents.dm]: Doesn't this code cause you to accumulate damp blackpowder very fast while in a blackpowder cloud or other sustained source? Look to me like it turns quantities of even 0.01 blackpowder into 5 dampblackpowder.

**Comment by XDTM** [comment]
Morphine can't be made nor can bar chem machines, neurotox is actually not that simple to make. Firesuits, biosuits, hardsuits block syringes.
Instant stuns are fine with limited availability/reliability, and the alternative is stacking poisons for an instakill anyway. At least with neurotox traitors have the option to kidnap without killing.

File: code/modules/reagents/chemistry/reagents/pyrotechnic_reagents.dm (hunk 1 above)
Comment (banthebantz) [inline code/modules/reagents/chemistry/reagents/pyrotechnic_reagents.dm]: YOU HAVE NO IDEA WHAT THE FUCK YOU ARE DOING AHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHH

**Comment by banthebantz** [comment]
>makes blackpowder a losebreath super death chem (lol guys its nerf!)
>makes neurotoxin inferior to any other stun or sleep chem

no

File: code/modules/reagents/chemistry/reagents/pyrotechnic_reagents.dm (hunk 1 above)
Comment (Iamgoofball) [inline code/modules/reagents/chemistry/reagents/pyrotechnic_reagents.dm]: Why do you think the metab rate is so low?

Comment (banthebantz) [inline code/modules/reagents/chemistry/reagents/pyrotechnic_reagents.dm]: i'm not talking to you goof

**Comment by kevinz000** [comment]
@LanCartwright iron is more common than morphine atleast morphine is limited without emagging

File: code/modules/reagents/chemistry/reagents/pyrotechnic_reagents.dm [hunk 5]
```diff
@@ -106,13 +106,31 @@
...
+
 /datum/reagent/blackpowder/on_ex_act()
 	var/location = get_turf(holder.my_atom)
 	var/datum/effect_system/reagents_explosion/e = new()
 	e.set_up(1 + round(volume/6, 1), location, 0, 0, message = 0)
 	e.start()
 	holder.clear_reagents()
 
+/datum/reagent/dampblackpowder
+	name = "Damp Black Powder"
```
Comment (MoreRobustThanYou) [inline code/modules/reagents/chemistry/reagents/pyrotechnic_reagents.dm]: https://www.youtube.com/watch?v=4ctK1aoWuqY

File: code/modules/reagents/chemistry/reagents/pyrotechnic_reagents.dm [hunk 6]
```diff
@@ -106,13 +106,31 @@
 	metabolization_rate = 0.05
 	taste_description = "salt"
 
+/datum/reagent/blackpowder/on_mob_life(mob/living/M)
+	M.losebreath += 10
```
Comment (MoreRobustThanYou) [inline code/modules/reagents/chemistry/reagents/pyrotechnic_reagents.dm]: > trying to nerf
> buffs black powder

File: code/modules/reagents/chemistry/reagents/alcohol_reagents.dm [hunk 7]
```diff
@@ -1162,10 +1162,10 @@ All effects don't start immediately, but rather get worse over time; the rate is
 	glass_desc = "A drink that is guaranteed to knock you silly."
 
 /datum/reagent/consumable/ethanol/neurotoxin/on_mob_life(mob/living/carbon/M)
-	M.Knockdown(60, 1, 0)
```
Comment (MoreRobustThanYou) [inline code/modules/reagents/chemistry/reagents/alcohol_reagents.dm]: no.

File: code/modules/reagents/chemistry/reagents/pyrotechnic_reagents.dm (hunk 2 above)
Comment (Supermichael777) [inline code/modules/reagents/chemistry/reagents/pyrotechnic_reagents.dm]: do if(prob(5))
about every 20 lifeticks

File: code/modules/reagents/chemistry/reagents/pyrotechnic_reagents.dm (hunk 4 above)
Comment (Supermichael777) [inline code/modules/reagents/chemistry/reagents/pyrotechnic_reagents.dm]: OH GOD WHY
fucking hell magic math

make it a blood catalyzed reaction or something jesus

File: code/modules/reagents/chemistry/reagents/alcohol_reagents.dm (hunk 3 above)
Comment (Supermichael777) [inline code/modules/reagents/chemistry/reagents/alcohol_reagents.dm]: yes

File: code/modules/reagents/chemistry/reagents/pyrotechnic_reagents.dm (hunk 4 above)
Comment (Supermichael777) [inline code/modules/reagents/chemistry/reagents/pyrotechnic_reagents.dm]: yes

File: code/modules/reagents/chemistry/reagents/pyrotechnic_reagents.dm (hunk 2 above)
Comment (LanCartwright) [inline code/modules/reagents/chemistry/reagents/pyrotechnic_reagents.dm]: alright

File: code/modules/reagents/chemistry/reagents/pyrotechnic_reagents.dm (hunk 4 above)
Comment (LanCartwright) [inline code/modules/reagents/chemistry/reagents/pyrotechnic_reagents.dm]: that's smoke shitcode

**Comment by LanCartwright** [comment]
@kevinz000 
weld wall down not an argument.


@XDTM 
Doesn't matter. Roundstart stunchems are gone. It's the reason why Zombie powder is so hard to get.

you have instastuns it's called Zombie powder. you also can kidnap easy with chloral and perflourodecalin or morphine and perfluorodecalin

File: code/modules/reagents/chemistry/reagents/pyrotechnic_reagents.dm (hunk 4 above)
Comment (CPTANT) [inline code/modules/reagents/chemistry/reagents/pyrotechnic_reagents.dm]: No its not, smoke just adds small amounts of reagents every tick. You code takes those small amounts and turns it into 5u at a time

File: code/modules/reagents/chemistry/reagents/pyrotechnic_reagents.dm (hunk 1 above)
Comment (Cyberboss) [inline code/modules/reagents/chemistry/reagents/pyrotechnic_reagents.dm]: Why are you checking if holder has blackpowder in on_mob_life? Is that not implied?

File: code/modules/reagents/chemistry/reagents/pyrotechnic_reagents.dm (hunk 4 above)
Comment (Cyberboss) [inline code/modules/reagents/chemistry/reagents/pyrotechnic_reagents.dm]: Rebalance this entire thing to use proper metabolization

File: code/modules/reagents/chemistry/reagents/pyrotechnic_reagents.dm (hunk 2 above)
Comment (Cyberboss) [inline code/modules/reagents/chemistry/reagents/pyrotechnic_reagents.dm]: Message spam

**Review by Cyberboss — CHANGES_REQUESTED** [review]

**Comment by Niknakflak** [comment]
Are permabanned people allowed to make pull requests

**Comment by LanCartwright** [comment]
@Niknakflak MRTY is underageB& and he's making PRs.

**Comment by Repukan** [comment]
Separation of coderbase and administration

**Comment by MoreRobustThanYou** [comment]
which is why kevinz usually doesn't mock me on irc

**Comment by Fox-McCloud** [comment]
the drink neurotoxin needs to be nerfed, in my opinion, but not these others.

**Comment by jughu** [comment]
Litteraly handholding
Dying is part of the game

**Comment by jughu** [comment]
Ok lets roast yout ass.

1.nuerotoxin stun is a joke i prefer to use spray tan for stuns anyways and its MUCH faster to make too.
2.seriously nerfing heparin? Making gauze more useless then it already is? Gauze exists for a reason if you get shot get some gauze put it on yourself then you try to get the chem out
3.blackpowder is a meme.

**Comment by jughu** [comment]
Also zombie powder was never meant to be a stun chem it is a chem tbat can fake your death to fool the oppoment

**Comment by Artemiuz28** [comment]
Hey. First attempting to nerf Perfluorodecalin, now this shit. Stop.

**Comment by PopNotes** [comment]
^

**Comment by MoreRobustThanYou** [comment]
^

**Comment by CPTANT** [comment]
"No more immediate and uncounterable stuns"

Yeah except tasers and stunbatons exist and can do that multiple times in a row.

**Comment by LanCartwright** [comment]
Due to a change in opinion, blackpowder is out of nerfs.

**Comment by Bawhoppen** [comment]
This is a textbook example of nerfing from personal experiences

**Comment by jughu** [comment]
>uncounterable stun
Thats called wearing a biosuit hardsuit to prevent to be syringe gunned

**Comment by PopNotes** [comment]
okay you're heading in the right direction, now all you have to do is repeat what you just did with black powder and do it to heparin and neurotoxin, then the PR will be perfect

**Comment by kevinz000** [comment]
@LanCartwright what?
wait wait what? iron's in the chem dispenser you don't need to weld walls.

**Comment by KingPhilipIII** [comment]
@jughu Wearing a hardsuit or a biosuit all the time is a terrible counter argument. Not only is it really gamey to be wearing that ALL the time, few players will take a reduction to movement speed in order to avoid being syringed at the cost of being far more vulnerable to every other threat.

**Comment by jughu** [comment]
Then git gud lol.

**Comment by CPTANT** [comment]
Dude git gud, Plenty of other stuns don't HAVE counters.

**Review by Cyberboss — APPROVED** [review]

**Comment by PopNotes** [comment]
cease

**Comment by optimumtact** [comment]
nobody will merge this with how unpopular it is

**Comment by ChangelingRain** [comment]
nobody.

**Comment by KorPhaeron** [comment]
Not in the mood to have the mob after me either, and people will just choose another chem after this

Honestly think @Iamgoofball is closer to the solution by looking at the syringe gun itself

**Comment by ChangelingRain** [comment]
The only reason goof was closer was that removing the syringe gun was @Ergovisavi's idea.

**Comment by Iamgoofball** [comment]
You're not proving me wrong

**Pull request closed without merging**